packages/sanity/src/core/releases/navbar/GlobalPerspectiveMenuItem.tsx

```diff
@@ -169,10 +174,10 @@ export const GlobalPerspectiveMenuItem = forwardRef<
               {displayTitle}
             </Text>
             {!isPublishedPerspective(release) &&
-              release.metadata.releaseType !== 'undecided' &&
+              release.metadata.releaseType === 'scheduled' &&
```

Comment:
Improvement to handle case that `intendedPublishAt` has not been updated when release type is (which shouldn't happen in studio, but could if CLI/API is used directly)